controllers/imagehub/api/v1/common_types.go

```diff
@@ -28,6 +31,19 @@ const (
 	defaultImagehubNamespace = "imagehub-system"
 )
 
+func GetSupportedKeywords() []string {
+	return []string{"Kubernetes", "Storage", "Network", "Database", "Monitoring", "Logging", "Dashboard", "MQ", "Platform", "GPU", "GitOps"}
+}
+
+var supportedKeywordsMap map[string]struct{}
```

Comment:
```go
var supportedKeywordsMap = func() map[string]struct{} {
return xxxx;
}()
```
Make map var simper at build time not init runtime.

Reply:
done.